tsconfig.json

```diff
@@ -183,6 +183,9 @@
       ],
       "@theia/ovsx-client/lib/*": [
         "dev-packages/ovsx-client/src/*"
+      ],
+      "@theia/dynamic-require/*": [
```

Comment:
I think you merged this package into core, right?

Reply:
Good catch, thought the reference generator would take care of these, but apparently not.